### PR TITLE
refactor: React accent-color-picker

### PR DESCRIPTION
**Pull request by Yserz**

### CODE REVIEW

File: src/script/components/accentColorPicker.ts [hunk 1]
```diff
@@ -48,7 +48,7 @@ class AccentColorPicker {
 }
 
 // Knockout registration of the accent color picker component.
-ko.components.register('accent-color-picker', {
+ko.components.register('BBBaccent-color-picker', {
```
Comment (lipis) [inline src/script/components/accentColorPicker.ts]: What's `BBB` means?

Comment (Yserz) [reply src/script/components/accentColorPicker.ts]: It's just excluded, will be removed completely when the PR is finished

File: src/script/util/test/TestPage.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  changeValue = (element: ReactWrapper, value: string) => this.do(() => element.simulate('change', {target: {value}}));
+  changeFiles = (element: ReactWrapper, files: File[]) => this.do(() => element.simulate('change', {target: {files}}));
+  submit = (element: ReactWrapper) => this.do(() => element.simulate('submit'));
+  mouseEnter = (element: ReactWrapper) => this.do(() => element.simulate('mouseenter'));
+  keyCodeUp = (element: ReactWrapper, keyCode: number) => this.do(() => element.simulate('keyup', {keyCode}));
+  keyCodeDown = (element: ReactWrapper, keyCode: number) => this.do(() => element.simulate('keydown', {keyCode}));
+
+  update = () => this.driver.update();
+  // eslint-disable-next-line no-console
+  debug = (element?: ReactWrapper) => console.log((element ? element : this.driver).debug());
```
Comment (lipis) [inline src/script/util/test/TestPage.tsx]: `console.info` maybe and remove the eslint-disable?

Comment (lipis) [inline src/script/util/test/TestPage.tsx]:
```suggestion
  debug = (element?: ReactWrapper) => console.info((element ? element : this.driver).debug());
```

File: src/script/util/test/TestPage.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  click = (element: ReactWrapper) => this.do(() => element.simulate('click'));
+  changeValue = (element: ReactWrapper, value: string) => this.do(() => element.simulate('change', {target: {value}}));
+  changeFiles = (element: ReactWrapper, files: File[]) => this.do(() => element.simulate('change', {target: {files}}));
+  submit = (element: ReactWrapper) => this.do(() => element.simulate('submit'));
+  mouseEnter = (element: ReactWrapper) => this.do(() => element.simulate('mouseenter'));
+  keyCodeUp = (element: ReactWrapper, keyCode: number) => this.do(() => element.simulate('keyup', {keyCode}));
+  keyCodeDown = (element: ReactWrapper, keyCode: number) => this.do(() => element.simulate('keydown', {keyCode}));
+
+  update = () => this.driver.update();
+  // eslint-disable-next-line no-console
```
Comment (Yserz) [inline src/script/util/test/TestPage.tsx:53]:
```suggestion
```

**Review by AndyLnd — APPROVED** [review]

**Pull request merged**